src/app.ts

```diff
@@ -977,12 +971,12 @@ app.get(
     spanWrapper(createGetAccountHandler(accountService, accountRepository)),
 );
 app.get(
-    '/.ghost/activitypub/posts',
+    '/.ghost/activitypub/posts/:handle',
     requireRole(GhostRole.Owner, GhostRole.Administrator),
-    spanWrapper(createGetAccountPostsHandler(accountService, postService)),
+    spanWrapper(createGetAccountPostsHandler(postService, accountRepository)),
 );
 app.get(
-    '/.ghost/activitypub/posts/liked',
+    '/.ghost/activitypub/posts/liked/:handle',
```

Comment:
Had to add the handle here as `posts/liked` was otherwise routing to `posts/:handle`

Reply:
Mentioned this on the FE PR, what about using `/posts/me/liked` instead?